context/app/routes_main.py

```diff
@@ -72,38 +72,6 @@ def ccf_eui():
     )
 
 
-@blueprint.route('/browse/collection/<uuid>')
-def collection_details(uuid):
-    client = _get_client()
-    collection = client.get_collection(uuid)
-    core_props = {'endpoints': _get_endpoints(), 'collection': collection}
-    template = f'pages/base_react.html'
-    return render_template(
-        template,
-        uuid=uuid,
-        title=f'{collection["display_doi"]} | Collection',
-        flask_data=core_props
-    )
-
-
-@blueprint.route('/browse/collection/<uuid>.<ext>')
-def collection_details_ext(uuid, ext):
-    if ext != 'json':
-        abort(404)
-    client = _get_client()
-
-    collection = client.get_collection(uuid)
-    return collection
-
-
-@blueprint.route('/browse/HBM<hbm_suffix>')
-def hbm_redirect(hbm_suffix):
-    client = _get_client()
-    entity = client.get_entity(hbm_id=f'HBM{hbm_suffix}')
-    return redirect(
-        url_for('routes.details', type=entity['entity_type'].lower(), uuid=entity['uuid']))
-
-
```

Comment:
Keep this one... Not much used, but at one point it was asked for.

Reply:
Whoops, not sure how that got in there, thanks!